packages/docusaurus-plugin-google-analytics/src/index.ts

```diff
@@ -40,7 +43,7 @@ module.exports = function (context) {
       if (!isProd) {
         return {};
       }
-      return {
+      const HTMLTags = {
```

Comment:
Don't understand, why create new const here, if we can return just object as it was before?

Reply:
That doesn't work and spits out a giant "Type not assignable "error"... My best guess is due to some inference funkiness